cmd/version.go

```diff
@@ -17,7 +20,7 @@ func newVersionCmd() *cobra.Command {
 		Long:  "Print the version of ACS-Engine",
 
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Infof("ACS-Engine Version: %s", BuildSHA)
+			log.Infof("ACS-Engine Version: %s (%s)", BuildTag, BuildSHA)
```

Comment:
Yeah. I considered that... I do like the machine-parseability of that style. I can adjust.